pkg/conf/conf_test.go

```diff
@@ -494,6 +494,7 @@ func TestEmbeddedExporter(t *testing.T) {
 	}
 
 	uniqueExporters := GetUniqueExporters(p.Exporters)
+	slices.Sort(uniqueExporters)
```

Comment:
no longer needed since GetUniqueExporters sorts the slice

Reply:
removed.